File: Baekjoon_Online_Judge/NO_15726/15726.cpp
#include <iostream>
using namespace std;
/*
 ---------------------------- 
 | Created by happiness96   | 
 | Year 2020                | 
 | Month 07                 | 
 | Day 14                   | 
 | 15726 이칙연산            | 
 ---------------------------- 
*/

int main(){
    double A, B, C, res1, res2;

    cin >> A >> B >> C;

    res1 = A * B / C;
    res2 = A / B * C;

    if(res1 > res2) cout << (int)res1;
    else cout << (int)res2;

    return 0;
}

File: Baekjoon_Online_Judge/NO_15726/15726.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# ---------------------------- #
# | Created by happiness96   | #
# | Year 2020                | #
# | Month 07                 | #
# | Day 13                   | #
# | 15726 이칙연산            | #
# ---------------------------- #


if __name__ == "__main__":
    A, B, C = map(int, r_input().split())

    print(int(max(A * B / C, A / B * C)))
